src/screens/auth/registration/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import AuthLayout from '../auth-layout';
+import RegistrationForm from './registration-form';
+import { Toast } from 'native-base';
+import { AsyncError } from '../../../types';
+
+const Registration: React.FC = () => {
```

Comment:
The name is not a noun, but sounds like a verb and makes a poor case to be used for a component.